.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
This is for [jEnv](https://github.com/gcuisinier/jenv), which I use to manage multiple JDKs. Should be harmless to anyone who doesn't.